src/amp-story-player/amp-story-player-impl.js

```diff
@@ -546,11 +556,12 @@ export class AmpStoryPlayer {
             );
           });
 
-          const customControls =
-            this.playerConfig_ && this.playerConfig_['controls'];
-
-          if (customControls) {
-            messaging.sendRequest('customDocumentUI', customControls, false);
+          if (this.playerConfig_) {
+            messaging.sendRequest(
+              'customDocumentUI',
+              /** @type {!JsonObject} */ (this.playerConfig_),
```

Comment:
This is going to sound very nitpicky but it'll be easier to maintain:
`this.playerConfig_` should contain all the config passed by the publisher, and here you should create a new JsonObject `{"controls": this.playerConfig_.controls}`.

Two benefits:
- When you'll add the `behavior` key to the config you'll have to do this change anyway
- You don't really need to pass the full player config to the story anyways, just part of it (`controls`)

Reply:
Yeah I was debating whether I should do this. But the thing is, when performing an action that belongs to the `behavior` key (like fetching stories / circular wrapping), we will not be reading those in this method (`setUpMessagingFromIframe()`). We are just dealing with the config in this method because we need to send it as a message to every story. But for the `behavior` key, we will need to access it only at the player level.

By creating the object (with `behavior` and `controls` keys) at the `buildCallback()`, we won't be spreading the addition of new keys in different parts of the code. WDYT?

Reply:
(Discussed offline and we think it's better passing just the `controls` key here since we don't want to send all the configuration, including the `behavior`).